.github/workflows/image-builder-golang.yml

```diff
@@ -0,0 +1,64 @@
+name: builder_image_golang_pipeline
```

Comment:
I'm thinking that adding `_pipeline` does not provide any new information to the name. Is this necessary?

Reply:
AFAIK no, thx, will remove them.